src/main/java/io/temporal/samples/hello/HelloActivity.java

```diff
@@ -33,34 +33,64 @@
 import java.time.Duration;
 
 /**
- * Hello World Temporal workflow that executes a single activity. Requires a local instance the
- * Temporal service to be running.
+ * Sample Temporal workflow that executes a single workflow activity.
+ *
+ * <p>To execute this example a locally running Temporal service instance is required. You can
+ * follow instructions on how to set up your Temporal service here:
+ * https://github.com/temporalio/temporal/blob/master/README.md#download-and-start-temporal-server-locally
  */
 public class HelloActivity {
 
-  static final String TASK_QUEUE = "HelloActivity";
+  // Define the task queue name
+  static final String TASK_QUEUE = "HelloActivityTaskQueue";
+
+  // Define our workflow unique id
+  static final String WORKFLOW_ID = "HelloActivityWorkflow";
 
-  /** Workflow interface has to have at least one method annotated with @WorkflowMethod. */
+  /**
+   * Define the Workflow Interface. It must contain at least one method annotated
```

Comment:
I believe it must have one and only one workflow method. Please double check, but I don't think we allow having multiple.

Reply:
+1 yes double checked only one WorkflowMethod annotated is allowed. Will fix everywhere